go/api/set_commands.go

```diff
@@ -278,4 +278,27 @@ type SetCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/spop/
 	SPop(key string) (Result[string], error)
+
+	// SUnionStore stores the members of the union of all given sets specified by `keys` into a new set at `destination`.
+	//
+	// Note: When in cluster mode, `destination` and all `keys` must map to the same hash slot.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//	 destination - The key of the destination set.
+	//   keys - The keys from which to retrieve the set members.
+	//
+	// Return value:
+	//   The number of elements in the resulting set.
+	//
+	// Example:
+	//   result, err := client.SUnionStore("my_set", []string{"set1", "set2"})
+	//   if err != nil {
+	//       fmt.Println(result)
```

Comment:
`result` or `result.Value()`?
```suggestion
	//       fmt.Println(result.Value())
```